### ISSUE TITLE
Modèle comprenant plusieurs sorties

### ISSUE THREAD

**Issue by Waltharnack** [Waltharnack/vietnam_coffee_detection#47]
une sortie softmax qui détermine le label, une autre en parallèle qui classe dans un groupe (agriculture / forêts / urbain ...)

Voir : [https://www.pyimagesearch.com/2018/06/04/keras-multiple-outputs-and-multiple-losses/](https://www.pyimagesearch.com/2018/06/04/keras-multiple-outputs-and-multiple-losses/)
